packages/nc-gui/components/project/spreadsheet/rowsXcDataTable.vue

```diff
@@ -1246,6 +1248,8 @@ export default {
           this.expandRow(data.length - 1, rowMeta)
         }
       }
+      await this.save()
```

Comment:
This will break the existing behaviour. 

Supposing users create some empty rows (with no input) and refresh the page, those empty rows will not be saved. However, if you add `await this.save()` here, it would save those empty rows as well which is not expected.

Therefore, you don't need this line. Please delete it.